unattended_installer/install_functions/installCommon.sh

```diff
@@ -248,8 +271,8 @@ function installCommon_installPrerequisites() {
             common_logger "--- Dependencies ----"
             for dep in "${not_installed[@]}"; do
                 common_logger "Installing $dep."
-                eval "DEBIAN_FRONTEND=noninteractive apt install ${dep} -y ${debug}"
-                if [  "$?" != 0  ]; then
+                installCommon_aptInstall ${dep}
+                    if [  "$?" != 0  ]; then
```

Comment:
Remove this indentation

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1401/commits/fccfc5546477826ac31928bc1caf68c0531d9976